index.html

```diff
@@ -2,10 +2,25 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./style.css" />
     <script src="index.js" defer></script>
     <title>Black Jack Game</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app" class="container">
+      <button href="#" class="shuffle">Shuffle</button>
```

Comment:
Buttons don't have an `href` they have a type which usually is set to `button` like so
```suggestion
      <button type="button" class="shuffle">Shuffle</button>
```